README.md

```diff
@@ -1,29 +1,102 @@
 # testcafe-aurelia-selectors
 
-[Tescafe](https://github.com/DevExpress/testcafe) selector extentions for [Aurelia](https://github.com/aurelia/framework) apps.
-
-## Avalible selector extentions
-- byValueBind
-- byShowBind
-- byCheckedBind
-- byFocusBind
-- byDbClickDelegate
-- byClickDelegate
+This plugin provides selector extensions that make it easier to test Aurelia applications with [TestCafe](https://github.com/DevExpress/testcafe/). These extensions allow you to select page elements in a way that is native to Aurelia.
 
 ## Install
-`npm install testcafe-aurelia-selectors`
 
-## Usage
+```
+npm install testcafe-aurelia-selectors
+```
+
+## Selector extensions
+
+### byValueBind
+
+Selects an element whose `value` attribute is bound to the specified model property.
+
+```
+byValueBind(model, selector)

```

Comment:
Let's rename the first parameter to `expression`.
See http://aurelia.io/hub.html#/doc/article/aurelia/binding/latest/binding-basics/1